README.md

```diff
@@ -564,7 +564,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
 ## CI and CD
 
   * [AccessLint](https://github.com/marketplace/accesslint) — AccessLint brings automated web accessibility testing into your development workflow. It's free for open source and education purposes.
-  * [appcircle.io](https://appcircle.io) — Automated mobile CI/CD/CT for iOS and Android with online emulators. 20-minute build timeout (60 minutes for Open Source) with single concurrency for free.
+  * [appcircle.io](https://appcircle.io) — An enterprise-grade mobile DevOps platform that automates the build, test, and publish store of mobile apps for faster, efficient release cycle
```

Comment:
Please include details of what is free

Reply:
@ripienaar thanks for your feedback, I updated the description. You can review again. Thank you very much for your time and interest.